### PR TITLE
Replace showToast to use showSnackbar

### PR DESCRIPTION
**Pull request by sherrif10**
## Requirements

- [ ] This PR has a title that briefly describes the work done including the ticket number. If there is a ticket, make sure your PR title includes a [conventional commit](https://o3-dev.docs.openmrs.org/#/getting_started/contributing?id=your-pr-title-should-indicate-the-type-of-change-it-is) label. See existing PR titles for inspiration.
- [ ] My work conforms to the [OpenMRS 3.0 Styleguide](https://om.rs/styleguide) and [design documentation](https://zeroheight.com/23a080e38/p/880723-introduction).
- [ ] My work includes tests or is validated by existing tests.

## Other

This PR replaces showToast with showSnackbar

Before
Screenshot from 2024-01-21 22-33-20
After
Screenshot from 2024-01-21 22-26-09

cc @pirupius

### CODE REVIEW

**Review by jabahum — APPROVED** [review]
Thanks @sherrif10

File: package.json [hunk 1]
```diff
@@ -1,123 +1,123 @@
...
+    "dayjs": "1.x",
+    "react": "18.x",
+    "react-i18next": "11.x",
+    "react-router-dom": "6.x",
+    "rxjs": "6.x"
+  },
+  "devDependencies": {
+    "@ohri/openmrs-esm-ohri-commons-lib": "next",
+    "@openmrs/esm-extensions": "next",
+    "@openmrs/esm-framework": "^5.3.2",
```
Comment (pirupius) [inline package.json:66]: Was this intentional? or did yarn do this? typically we use `next` tag so even after the `yarn up` it's better to leave it as `next`

File: package.json [hunk 2]
```diff
@@ -1,123 +1,123 @@
...
+    "eslint-config-ts-react-important-stuff": "^3.0.0",
+    "eslint-plugin-prettier": "^4.2.1",
+    "file-saver": "^2.0.5",
+    "husky": "^8.0.0",
+    "identity-obj-proxy": "^3.0.0",
+    "jest": "^28.1.3",
+    "jest-cli": "^28.1.3",
+    "jest-environment-jsdom": "^28.1.3",
+    "lerna": "^5.6.1",
+    "openmrs": "^5.3.2",
```
Comment (pirupius) [inline package.json:99]: same applies here

File: package.json (hunk 1 above)
Comment (sherrif10) [reply package.json:66]: Yarn did this. Let me get it back

**Review by pirupius — APPROVED** [review]
Thanks @sherrif10

**Pull request merged**